packages/shared/src/components/Feed.tsx

```diff
@@ -357,17 +348,17 @@ export default function Feed<T>({
     });
 
   const onShareClick = (post: Post, row?: number, column?: number) =>
-    openSharePost(post, virtualizedNumCards, column, row);
+    openSharePost({ post, columns: virtualizedNumCards, column, row });
 
   const post = (items[postMenuIndex] as PostItem)?.post;
   const commonMenuItems = {
     onShare: () =>
-      openSharePost(
+      openSharePost({
         post,
-        virtualizedNumCards,
-        postMenuLocation.row,
-        postMenuLocation.column,
-      ),
+        columns: virtualizedNumCards,
+        row: postMenuLocation.row,
```

Comment:
One of the benefits of using an object is this: the row was on the 3rd parameter, but it is supposed to be the column.